supplementary_style_guide/glossary_terms_conventions/general_conventions/t.adoc

```diff
@@ -302,7 +302,7 @@ A web server uses its public key to obtain a certificate from a trusted CA. The
 [discrete]
 [[truth-maintenance-system]]
 ==== image:images/yes.png[yes] truth maintenance system (noun)
-*Description*: In Red Hat JBoss BRMS and Red Hat JBoss BPM Suite, a "truth maintenance system" (TMS) refers to the ability of the inference engine to enforce truthfulness when applying rules. The truth maintenance system uses the mechanism of truth maintenance to efficiently handle the inferred information from rules. It provides justified reasoning for each and every action taken by the inference engine and validates the conclusions of the engine. If the inference engine asserts data as a result of firing a rule, the engine uses the truth maintenance to justify the assertion.
+*Description*: In Red Hat JBoss BRMS and Red Hat JBoss BPM Suite, a _truth maintenance system (TMS)_ refers to the ability of the inference engine to enforce truthfulness when applying rules. The truth maintenance system uses the mechanism of truth maintenance to efficiently handle the inferred information from rules. It provides justified reasoning for each and every action taken by the inference engine and validates the conclusions of the engine. If the inference engine asserts data as a result of firing a rule, the engine uses the truth maintenance to justify the assertion.
```

Comment:
```suggestion
*Description*: In Red Hat JBoss BRMS and Red Hat JBoss BPM Suite, a _truth maintenance system (TMS)_ refers to the ability of the inference engine to enforce truthfulness when applying rules. The `truth maintenance` system uses the mechanism of truth maintenance to efficiently handle the inferred information from rules. It provides justified reasoning for each and every action taken by the inference engine and validates the conclusions of the engine. If the inference engine asserts data as a result of firing a rule, the engine uses the truth maintenance to justify the assertion.
```

Reply:
@dfitzmau Are you sure about this one? I searched the docs for this and in this doc that came up, I don't see "truth maintenance" in backticks at all: https://access.redhat.com/documentation/en-us/red_hat_process_automation_manager/7.7/html/decision_engine_in_red_hat_process_automation_manager/inference-and-truth-maintenance_decision-engine

(And same question as above comment - this is "system", not an EAP "subsystem")